yellowbrick/datasets/manifest.json

```diff
@@ -31,6 +31,10 @@
     "url": "https://s3.amazonaws.com/ddl-data-lake/yellowbrick/v1.0/occupancy.zip",
     "signature": "0b390387584586a05f45c7da610fdaaf8922c5954834f323ae349137394e6253"
   },
+  "receivers": {
+    "url": "receivers.zip",
+    "signature": "a5dbe22d267e9aa12bd84be325c417c27882d156d4410a718262aad6e235ab9b"
+  },
```

Comment:
This should now be:

```
"nfl": {
    "url": "https://s3.amazonaws.com/ddl-data-lake/yellowbrick/v1.0/nfl.zip",
    "signature": "e2d0665ed4fd7bea866ba6d2f0035fce89e6143da3c12dcd870e67319451a382"
  }
```

I've updated yellowbrick-datasets with the new manifest if you just want to copy it over.